.config/forest.dic

```diff
@@ -42,29 +46,41 @@ mainnet
 MDNS
 mempool
 Merkle
+METADATA_FILE_SUFFIX
 MiB
 milliGas
 multihash
 multisig
 mutex
 P2P
 parsable
+parse_filename
 pointer/SM
 PoSt
+recognised
 RPC
 SECP
 SECP256k1
 serializable
 serializer/SM
+snapshot_dir
```

Comment:
In general, you can avoid putting those in the dictionary by surrounding them with backticks. This would allow removing several entries you've added here.